src/main/java/com/maxmind/geoip2/DatabaseReader.java

```diff
@@ -409,6 +413,51 @@ private Optional<AnonymousIpResponse> getAnonymousIp(
                 )
         );
     }
+    /**
```

Comment:
There should be a emtpy line between methods.

Reply:
I added a new line but for some reason github doesn't show this as outdated.